services/user/XAdmin/ui/src/components/forms/chain-mode-form.tsx

```diff
@@ -21,6 +21,7 @@ interface Props {
 
 export const ChainTypeForm = ({ form, next }: Props) => {
     const isSecure = window.location.protocol === "https:";
+    const isLocalhost = window.location.hostname.endsWith("localhost");
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/isSecureContext

Reply:
Ah, I didn't know about this. Thank you!